Source/items.cpp

```diff
@@ -1492,6 +1500,7 @@ int RndTypeItems(int itype, int imid, int lvl)
 
 _unique_items CheckUnique(int i, int lvl, int uper, bool recreate)
 {
+	auto &item = Items[i];

```

Comment:
If there is just a single use in the function isn't not really beneficial do do this